app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx

```diff
@@ -32,13 +32,15 @@ export const ListingDetailsAdditionalInformation = ({
         {listing.Listing_Other_Notes && (
           <div className="info-card bg-gray-100 border-0">
             <h3 className="text-serif-lg">{t("listings.specialNotes")}</h3>
-            <TextTruncate text={listing.Listing_Other_Notes} />
+            <div className="text-sm">
```

Comment:
I think TextTruncate already wraps the jsx in a div with text-sm. The [snapshot](app/javascript/__tests__/modules/listingDetails/__snapshots__/ListingDetailsAdditionalInformation.test.tsx.snap) shows a div with a text-sm class wrapping a div with a text-sm class. Not sure if that's intended, just something I noticed!